front-end/src/app/shared/services/api.service.ts

```diff
@@ -24,7 +24,7 @@ export class ApiService {
   getHeaders(headersToAdd: object = {}) {
     const csrfToken = `${this.cookieService.get('csrftoken')}`;
     const baseHeaders = {
-      'Content-Type': 'application/json',
+      //'Content-Type': 'application/json',
```

Comment:
Specifying the Content Type header was preventing angular from determining the header dynamically, which was preventing it from setting a properly formatted multipart-form content type.